apps/company/serializers.py

```
@@ -193,53 +225,81 @@ class Meta:
-                    results[vendor_code][date] = order.ozon_quantity + order.wildberries_quantity + order.yandex_market_quantity
 
-        return results
+        if (date_to - date_from).days == 0:
+            date = date_from
+            date_range = queryset.values("warehouse")
+            for warehouse in date_range:
+                day_sales = queryset.filter(date__date=date, warehouse=warehouse).values('product__vendor_code').annotate(total_sales=Count('id'))
+                for sale in day_sales:
+                    product_code = sale['product__vendor_code']
+                    if product_code not in results:
+                        results[product_code] = { warehouse.oblast_okrug_name: 0 for date in date_range}
```

Ruff

Dictionary comprehension uses static key: `warehouse.oblast_okrug_name`

(B035)